ext/ddtrace_profiling_native_extension/collectors_cpu_and_wall_time.c

```diff
@@ -147,7 +189,6 @@ static VALUE _native_new(VALUE klass) {
    // "numtable" is an awful name, but TL;DR it's what should be used when keys are `VALUE`s.
     st_init_numtable();
   state->recorder_instance = Qnil;
-  state->sample_count = 0;
```

Comment:
Is this initialized somewhere else now?

Reply:
Good question!

Since the memory is allocated with `calloc` (more specifically, `ruby_xcalloc`, on line 184), it's guaranteed to be set to zero.

I was torn between adding initialization to all the new stats (`gc_samples`, `gc_samples_missed_due_to_missing_context`, `gc_samples_missed_due_to_missing_sample_after_gc`) as well, and not having it at all, since it's not needed.

I decided to not have the explicit initialization. to hopefully reduce some of the boilerplate, but let me know if this is confusing.